src/app/app.module.ts

```diff
@@ -61,8 +61,13 @@ export class AppModule {
     this.setLanguage();
   }
   setLanguage() {
-    this.translate.setDefaultLang('en');
-    this.translate.use('en'); 
+    let language = localStorage.getItem('preferred_language')
+    if (language) {
```

Comment:
what if there's no language stored, any reason we are not handling the else part like default setting?

Reply:
there will be by default english from the backend what will be getting stored

Reply:
When will that be done